wallmatrix/sources/mbta.py

```diff
@@ -7,31 +8,41 @@
 from wallmatrix.sources import Source
 
 
+class Line(TypedDict):
+    color: str
+    line: str
+    route: str
+    origin: str
+    destination: str
+    direction: int
+    walk_min: int
+
+
 class MBTA(Source):
     SOURCE_NAME = "MBTA North from NEU"
 
-    LINES = [
+    LINES: list[Line] = [
```

Comment:
We may want to put the green line above the other line, since it's the one we take most frequently.